src/components/stepindicator/StepIndicator/StepIndicator.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import classnames from 'classnames'
+import { StepIndicatorStepProps } from '../StepIndicatorStep/StepIndicatorStep'
+
+interface StepIndicatorProps {
+  showLabels?: boolean
+  counters?: true | false | 'small'
```

Comment:
I think this type is pretty confusing because its a boolean or a string. Lets make this more similar to our variant props which use string union type -  `default | small  | none`  or something that?

Reply:
fixed in ec8cf08